src/js/components/FileInput/stories/Custom.js

```diff
@@ -5,6 +5,13 @@ import { Trash } from 'grommet-icons';
 
 const customTheme = {
   fileInput: {
+    button: {
+      pad: {
+        vertical: '4px',
+        horizontal: '8px',
+      },
+      default: {},
```

Comment:
I don't think this is necessary for the styling to occur. maybe we add some background color or border to the button to make the custom styling abilities more apparent?